x-pack/plugins/canvas/public/components/sidebar/sidebar_section_title.js

```diff
@@ -10,7 +10,7 @@ import { EuiTitle, EuiFlexItem, EuiFlexGroup, EuiToolTip } from '@elastic/eui';
 
 export const SidebarSectionTitle = ({ title, tip, children }) => {
   const formattedTitle = (
-    <EuiTitle size="xs">
+    <EuiTitle size="xxs">
```

Comment:
Was there a reason you made the titles smaller? It still has the same height, the text is just smaller now...

Reply:
I made the main sidebar title to fit the new, longer titles and the additional action buttons. To keep the typographic hierarchy in order, I chose to then make the panel titles one size smaller. Also in this area, I added a slight bit of additional padding in the sidebar title are so that it aligned vertically with the workpad menu items (Add an element, etc.).